client/src/bg/background.js

```diff
@@ -4,10 +4,51 @@
 //     "sample_setting": "This is how you use Store.js to remember values"
 // });
 
+function generateUUID() {
+  let id = 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, function(c) {
+    let r = Math.random()*16|0, v = c === 'x' ? r : (r&0x3|0x8);
+    return v.toString(16);
+  });
+  return id;
+}
```

Comment:
I think right now this serves our purpose and allows us to move on. Using Chrome's user identify involving touch more areas, and this UUID is good unless we have millions of users. We can make the switch later and be agiler right now.

There are others who think the same: https://stackoverflow.com/questions/23822170/getting-unique-clientid-from-chrome-extension#answer-23854032